cargo-quickinstall/src/args.rs

```diff
@@ -10,6 +10,7 @@ OPTIONS:
         --version <VERSION>         Specify a version to install
         --target <TRIPLE>           Install package for the target triple
         --no-fallback               Don't fall back to `cargo install`
+        --dry-run                   Print the \"curl | tar\" command that would be run to fetch the binary
     -V, --print-version             Print version info and exit
     -h, --help                      Prints help information
 ";
```

Comment:
if you turn this string constant into a "raw" string, like
```
pub const HELP: &str = r#"USAGE:
...
"#;
```
then you don't need to escape the quotes. Alternatively, use backticks, like the line above does.

Reply:
I am going with the easy option, the backtick :)